scripts/zones/Abyssea-Tahrongi/MobIDs.lua

```diff
@@ -0,0 +1,28 @@
+require("scripts/globals/keyitems");
+
+-- [QM] = {'name', {list of required trades}, {list of required KIs}, nmID}
+-- uncomment lines as NMs are implemented
+NM_POPS = 
+{
+--  [16961954] = { 'qm1',           {2915},                                                                                         {}, 16961917}, -- Halimede
+--  [16961955] = { 'qm2',           {2916},                                                                                         {}, 16961918}, -- Vetehinen
+--  [16961956] = { 'qm3', {2917,2945,2946},                                                                                         {}, 16961919}, -- Ophanim
+--  [16961957] = { 'qm4',           {2918},                                                                                         {}, 16961920}, -- Cannered Noz
+--  [16961958] = { 'qm5',           {2919},                                                                                         {}, 16961921}, -- Treble Noctules
```

Comment:
http://ffxiclopedia.wikia.com/wiki/Treble_Noctules
requires Exorcised Skull & Bloody Fang in trade